gui/default/syncthing/core/syncthingController.js

```diff
@@ -1402,6 +1402,11 @@ angular.module('syncthing.core')
                 autoNormalize: true
             };
             $scope.editingExisting = false;
+            $scope.newIgnores = {
+                edited: false,
```

Comment:
No need to track this. If content is not '', you can always set the patterns regardless if it was edited or not, don't see any harm. I guess you can skip if it's empty string.